### DISCUSSION TITLE
New areas affected by lag compensation in patch 1037-Lag-compensation-ticks.patch

### DISCUSSION THREAD

**Discussion by wicivo** [PaperMC/Paper#11060] [Ideas]
### Is your feature request related to a problem?

Add new areas affected by lag compensation

### Describe the solution you'd like.

Add new areas affected by lag compensation, such as these:

Item pickup delay
Block entity tick acceleration
Portal delay
Sleeping delay
Potion delay
Fluid spread speed

### Describe alternatives you've considered.

https://github.com/snackbag/tt20

**Comment by ghost** [comment]
idk how to feel about lag compensation tbh, it would be better to fix the underlying lag problem by optimizing config/plugin or if you have many players using multiple servers or folia

**Comment by ComboGame** [comment]
like

**Comment by DevPixelP** [comment]
Очень сильно нравиться
